pkg/libvirttools/virtualization.go

```diff
@@ -167,7 +167,6 @@ func (v *VirtualizationTool) processVolumes(containerName string, mounts []*kube
 				return copyDomXML, err
 			}
 			domXML = string(outArr[:])
-			glog.Infof("Creating domain:\n%s", domXML)
```

Comment:
It's not removed, it's moved into (IMO) better suited place later.
https://github.com/Mirantis/virtlet/pull/73/files/cb3ae668c0cba5a93e92ca425caa6fceb4759ef5#diff-27a11a3eecdfb4f023d0be5eb326dee3R265

It's about container creation, not about volume processing.